app/mailers/base_mailer.rb

```diff
@@ -27,4 +27,21 @@ def email_subject_prefix(group_name)
   def from_user_via_loomio(user)
     "\"#{user.name} (Loomio)\" <notifications@loomio.org>"
   end
+
+  def send_single_mail(locale: I18n.locale, to:, subject:, **options)
+    I18n.with_locale(locale) { mail options.merge(to: self.class.email_interceptor || to, subject: subject) }
+  end
+
+  def self.send_bulk_mail(to:)
+    to = Array(User.loomio_helper_bot) if email_interceptor.present?
```

Comment:
This is a bit weird; it's because the blocks passed to `send_bulk_mail` always call `send_single_mail` under the hood, so I'm less concerned with who the recipients are (because all of them will be sent to the interceptor anyway), and more concerned with the number of recipients (we want one so that starting a thread in a group doesn't clog the inbox with 100 emails.)